libs/shared/src/config/contextPath.ts

```diff
@@ -8,29 +8,41 @@ export enum NovuComponentEnum {
 export function getContextPath(component: NovuComponentEnum) {
   let contextPath = '';
 
-  if (process.env.GLOBAL_CONTEXT_PATH) {
-    contextPath += process.env.GLOBAL_CONTEXT_PATH + '/';
+  // Determine if we are running in the browser or in node.js. If we are
+  // running in node.js, we will have access to the process.env object,
+  // otherwise we will have access to the window._env_ object to get the
+  // environment variables.
```

Comment:
Seems like there is some eslint error here, i believe you would need to convert this to a multiline comment `/**`

Reply:
Thanks. I updated the code.